ivy/functional/frontends/tensorflow/keras/layers.py

```diff
@@ -0,0 +1,15 @@
+import ivy
+import ivy.functional.frontends.tensorflow as tf_frontend
+from ivy.functional.frontends.tensorflow.func_wrapper import to_ivy_arrays_and_back
+
+@to_ivy_arrays_and_back
+def flatten(x,/,*,copy=None,start_dim=0, end_dim=-1, order='C', out=None):
```

Comment:
I tooj help from link:
https://unify.ai/docs/ivy/docs/functional/ivy/experimental/manipulation/ivy.functional.ivy.experimental.manipulation.flatten.html

Reply:
Ah, so the idea of the frontend functions is that they copy the structure of the frontend framework (in this case tensorflow) and internally call the ivy function. So rather than using the call signature for `ivy.flatten`, you should be using the call signature for `tensorflow.keras.flatten` and then calling the ivy function from within your function. Does that make sense?

Reply:
yes I understood,thanks!!